test/parallel/test-buffer-writefloat.js

```diff
@@ -50,8 +50,18 @@ assert.strictEqual(buffer.readFloatLE(4), -Infinity);
 
 buffer.writeFloatBE(NaN, 0);
 buffer.writeFloatLE(NaN, 4);
-assert.ok(buffer.equals(
-  new Uint8Array([ 0x7F, 0xC0, 0x00, 0x00, 0x00, 0x00, 0xC0, 0x7F ])));
+
+// JS only knows a single NaN but there exist two platform specific
+// implementations. Therefore, allow both quiet and signalling NaNs.
+if (buffer[1] === 0xF7) {
+  assert.ok(
+    buffer.equals(new Uint8Array(
+      [ 0x7F, 0xBF, 0xFF, 0xFF, 0xFF, 0xFF, 0xBF, 0x7F ])));
+} else {
+  assert.ok(
+    buffer.equals(new Uint8Array(
+      [ 0x7F, 0xC0, 0x00, 0x00, 0x00, 0x00, 0xC0, 0x7F ])));
+}
```

Comment:
`buffer[1] === 0xF7` isn't true for either assert?

Reply:
It was late ^^. Fixed.